support-frontend/app/controllers/Promotions.scala

```diff
@@ -40,6 +40,7 @@ class Promotions(
         case GuardianWeekly => routes.WeeklySubscription.weeklyGeoRedirect(promotionTerms.isGift).url
         case DigitalPack => routes.DigitalSubscriptionController.digitalGeoRedirect().url
         case Paper => routes.PaperSubscription.paper(false).url
+        case Contribution => routes.Application.contributeGeoRedirect("").url
```

Comment:
this was just a scalac warning that kept popping up due to bad types so I fixed it the easy way